jest.config.ts

```diff
@@ -1,6 +1,7 @@
 import type { InitialOptionsTsJest } from 'ts-jest/dist/types'
 
 const config: InitialOptionsTsJest = {
+  rootDir: './tests',
```

Comment:
This was causing the failing tests. Jest was trying to run the template repo's tests